esdt-safe/src/lib.rs

```diff
@@ -182,7 +194,7 @@ pub trait EsdtSafe {
         /* TODO: Uncomment once the integration is complete
         self.ethereum_fee_prepay_proxy(self.fee_estimator_contract_address().get())
```

Comment:
the gwei / eth price contract is ready and updated - please try if it works with that.

Reply:
Will try to get the integration ready tomorrow.